terraform/aptos-node-testnet/main.tf

```diff
@@ -65,7 +65,7 @@ module "genesis-helm-values-deepmerge" {
   # https://registry.terraform.io/modules/Invicton-Labs/deepmerge/null/0.1.5
   source = "Invicton-Labs/deepmerge/null"
   maps = [
-    var.enable_forge ? local.genesis_helm_values_forge_override : map(),
+    var.enable_forge ? local.genesis_helm_values_forge_override : {},
```

Comment:
does this need the `jsonencode({}))` like above as well?

Reply:
Good catch!